packages/terra-form-validation/package.json

```diff
@@ -3,7 +3,6 @@
   "main": "docs/README.md",
```

Comment:
Let's move [final-form](https://github.com/cerner/terra-framework/blob/e1ac6427b914cb570d40b4fc2d91279ef473c50d/packages/terra-form-validation/package.json#L29) and [react-final-form](https://github.com/cerner/terra-framework/blob/e1ac6427b914cb570d40b4fc2d91279ef473c50d/packages/terra-form-validation/package.json#L31) to be both a peerDependency as well as devDependency

Reply:
Added here https://github.com/cerner/terra-framework/pull/635/commits/8e55e43d8a7dcc9d77396ff2e45042877751fc94